### DISCUSSION TITLE
MultiZone Optimization

### DISCUSSION THREAD

**Discussion by z-g-h** [NAnand-TUD/parablade#13] [Q&A] [answered]
how to config optimzation.cfg and SU2_config to optimize a multizone blade ? e.g. stator and rotor simulation

**Comment by NAnand-TUD** [comment, marked as answer]
Hello! Thank you for your question. This is a feature that we have developed but is not part of the open-source package just yet. We, will keep you informed when it will be available.

**Reply by z-g-h** [reply]
Thanks for your reply. I have another question is that  how do I make my own shape of curve to extend the shroud and hub. It seems that the program in Blade3D is just  extending the tangent line at leading and trailing edge. I want to make my own shape of curve to extend the shroud and hub edge, because my search has other zone and I have to make interface to other surface.

**Reply by NAnand-TUD** [reply]
I would say in that case you would have to code it yourself in the meshing section of the code.
